tests/test_flows.py

```diff
@@ -384,22 +384,67 @@ def test_channel_selection(self):
 
         self.debug("Send channel selection request")
 
-        payload_wlan0="0x14 {0x51 {0x0C {0x01 0x02 0x03 0x04 0x05 0x07 0x08 0x09 0x0A 0x0B 0x0C 0x0D} 0x00}} {0x52 {0x00 0x00}} {0x53 {0x08 {0x01 0x02 0x03 0x04 0x05 0x07 0x08 0x09} 0x00}} {0x54 {0x08 {0x05 0x07 0x08 0x09 0x0A 0x0B 0x0C 0x0D} 0x00}} {0x73 {0x00 0x00}} {0x74 {0x00 0x00}} {0x75 {0x00 0x00}} {0x76 {0x00 0x00}} {0x77 {0x00 0x00}} {0x78 {0x00 0x00}} {0x79 {0x00 0x00}} {0x7A {0x00 0x00}} {0x7B {0x00 0x00}} {0x7C {0x00 0x00}} {0x7D {0x00 0x00}} {0x7E {0x00 0x00}} {0x7F {0x00 0x00}} {0x80 {0x00 0x00}} {0x81 {0x00 0x00}} {0x82 {0x00 0x00}}"
         payload_empty="0x14"
-        payload_wlan2="0x14 {0x51 {0x00 0x00}} {0x52 {0x00 0x00}} {0x53 {0x00 0x00}} {0x54 {0x00 0x00}} {0x73 0x03 {0x28 0x2C 0x30} 0x00} {0x74 0x01 {0x2C} 0x00} {0x75 {0x00 0x00}} {0x76 {0x00 0x00}} {0x77 {0x00 0x00}} {0x78 {0x00 0x00}} {0x79 {0x00 0x00}} {0x7A {0x00 0x00}} {0x7B {0x00 0x00}} {0x7C {0x00 0x00}} {0x7D {0x00 0x00}} {0x7E {0x00 0x00}} {0x7F {0x00 0x00}} {0x80 0x05 {0x3A 0x6A 0x7A 0x8A 0x9B} 0x00} {0x81 {0x00 0x00}} {0x82 {0x00 0x00}}"
-
-        self.gateway_ucc.dev_send_1905(self.mac_repeater1, 0x8006, \
-          (0x8B,0x005F,"{%s %s}" % (self.mac_repeater1_wlan0,payload_wlan0)), \
-          (0x8D,0x0007,"{%s %s}" % (self.mac_repeater1_wlan0,payload_empty)), \
-          (0x8B,0x004C,"{%s %s}" % (self.mac_repeater1_wlan2,payload_wlan2)), \
-          (0x8D,0x0007,"{%s %s}" % (self.mac_repeater1_wlan2,payload_empty)))
+        payload_wlan0=' '.join([
+            '0x14',
+            '{0x51 {0x0C {0x01 0x02 0x03 0x04 0x05 0x07 0x08 0x09 0x0A 0x0B 0x0C 0x0D} 0x00}}',
+            '{0x52 {0x00 0x00}}',
+            '{0x53 {0x08 {0x01 0x02 0x03 0x04 0x05 0x07 0x08 0x09} 0x00}}',
+            '{0x54 {0x08 {0x05 0x07 0x08 0x09 0x0A 0x0B 0x0C 0x0D} 0x00}}',
+            '{0x73 {0x00 0x00}}',
+            '{0x74 {0x00 0x00}}',
+            '{0x75 {0x00 0x00}}',
+            '{0x76 {0x00 0x00}}',
+            '{0x77 {0x00 0x00}}',
+            '{0x78 {0x00 0x00}}',
+            '{0x79 {0x00 0x00}}',
+            '{0x7A {0x00 0x00}}',
+            '{0x7B {0x00 0x00}}',
+            '{0x7C {0x00 0x00}}',
+            '{0x7D {0x00 0x00}}',
+            '{0x7E {0x00 0x00}}',
+            '{0x7F {0x00 0x00}}',
+            '{0x80 {0x00 0x00}}',
+            '{0x81 {0x00 0x00}}',
+            '{0x82 {0x00 0x00}}',
+        ])
+
+        payload_wlan2=' '.join([
```

Comment:
I don't understand this change - I get the ACK change, but not the payload. Please explain it better in the commit message and in a comment. Also, please keep in 1-3 lines, it doesn't help much to spread this over a full screen, especially since the values are just hex.